snapcraft.yaml

```diff
@@ -106,7 +106,7 @@ parts:
   alsa:
     plugin: nil
     # after: [alsa-plugins, alsa-lib]
-    prepare: |
+    override-build: |
```

Comment:
Please move the `cat > alsa.conf` block to a new `override-pull` instead.

Reply:
Shouldn't all heredocs blocks belongs to `override-pull`